### PR TITLE
fix: Auth env detection for capacitor

### PR DESCRIPTION
**Pull request by riderx**
No change to current codebase is just addition to make the SDK work as expected in capacitor app, like in Cordova app

### CODE REVIEW

File: packages/auth-compat/src/platform.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ function _isHttpOrHttps(): boolean {
  */
 export function _isAndroidOrIosCordovaScheme(ua: string = getUA()): boolean {
   return !!(
-    (_getCurrentScheme() === 'file:' || _getCurrentScheme() === 'ionic:') &&
+    (_getCurrentScheme() === 'file:' || _getCurrentScheme() === 'ionic:' || _getCurrentScheme() === 'capacitor:') &&
```
Comment (sam-gc) [inline packages/auth-compat/src/platform.ts:54]: Please run `yarn format` in the root of the repo (I think this will cause the line to wrap)

**Review by sam-gc — CHANGES_REQUESTED** [review]
Thanks for this PR. Have you verified this works in a capacitor project?

**Comment by riderx** [comment]
@sam-gc it's a solution was proposed and test in the issue https://github.com/firebase/firebase-js-sdk/issues/5020#issuecomment-1116237515

File: packages/auth-compat/src/platform.ts (hunk 1 above)
Comment (riderx) [reply packages/auth-compat/src/platform.ts]: Done :)

Comment (riderx) [reply packages/auth-compat/src/platform.ts]: Done

**Comment by riderx** [comment]
@avolkovi @lisajian @yuchenshi, i know you have ton of work but please take 5 min to review it.
That a tiny one who will unlock all capacitor/ionic devs and close complains :)

**Comment by sam-gc** [comment]
Hi @riderx, it looks like the formatting check failed. Please run `yarn format` and we'll rerun the analysis

**Comment by riderx** [comment]
@sam-gc i did `yarn lint` and `yarn format` and there no change.
The ci For formatter fail at the step `Check for changes (fail if so)` witch is normal no ?

**Comment by sam-gc** [comment]
@riderx yeah it appears that file is updated. I'm rerunning the status checks

**Comment by riderx** [comment]
@sam-gc ok i fixed with the feedback of the CI
 In local i have no issue i don't know why
Screenshot 2022-05-20 at 20 15 50

**Comment by riderx** [comment]
@sam-gc the test have failed again and i fixed again, i still don't know why i cannot reproduce in local, sorry about that.
If you can allow again all should be good now

**Comment by sam-gc** [comment]
Okay it's looking good, @riderx would you mind adding a changeset before mergning (see the comment up above for instructions).

**Comment by riderx** [comment]
Of course @sam-gc i will :)

**Comment by riderx** [comment]
@sam-gc done, i hope i understand it well :)

**Comment by sam-gc** [comment]
Looks great, thanks again! Once the workflow run completes please feel free to merge when the button is enabled

**Comment by riderx** [comment]
@sam-gc one CI is still failing it says the token is missing, so i can't merge myself

**Review by MichaelBurdett — APPROVED** [review]

**Comment by sam-gc** [comment]
Odd, I don't know why it's blocking you but I'll go ahead and merge.

**Pull request merged**